src/main.ts

```diff
@@ -71,6 +71,18 @@ export { type Board, BoardClient } from './components/board';
 export { default as cameraApi } from './gen/component/camera/v1/camera_pb';
 export { type Camera, CameraClient } from './components/camera';
 
+/**
+ * Raw Protobuf interfaces for an Encoder component.
+ *
+ * Generated with https://github.com/improbable-eng/grpc-web
+ *
+ * @deprecated Use {@link EncoderClient} instead.
+ * @alpha
+ * @group Raw Protobufs
+ */
+export { default as encoderApi } from './gen/component/encoder/v1/encoder_pb';
+export { type Encoder, EncoderClient } from './components/encoder';
```

Comment:
if you agree with re-exporting the type

```suggestion
export { type Encoder, type EncoderProperties, EncoderClient } from './components/encoder';
```